Raven.Studio.Html5/App/commands/database/cluster/extendRaftClusterCommand.ts

```diff
@@ -18,10 +18,6 @@ class extendRaftClusterCommand extends commandBase {
 
             var url = "/admin/cluster/join";
 
-            if (this.force) {
```

Comment:
The force parameter is no longer used, because we removed the kidnap option we no longer need to force joining